middleware/chi/go.mod

```diff
@@ -48,3 +48,5 @@ require (
 	google.golang.org/grpc v1.50.1 // indirect
 	google.golang.org/protobuf v1.28.1 // indirect
 )
+
+replace github.com/tel-io/instrumentation/middleware/http => ../http
```

Comment:
replace github.com/tel-io/instrumentation/middleware/http => ../http 
!!! remove this you break tag fetch

Reply:
done